<launch>

 <arg name="world_file" default="$(env TURTLEBOT_GAZEBO_WORLD_FILE)"/>
 <arg name="gui" default="true"/>


 <include file="$(find gazebo_ros)/launch/empty_world.launch">
   <arg name="use_sim_time" value="true"/>
   <arg name="debug" value="false"/>
   <arg name="gui" value="$(arg gui)" />
   <arg name="world_name" value="$(arg world_file)"/>
 </include>

 <group ns="turtle1">
  <param name="tf_prefix" value="turtle1" />
  <include file="$(find multirobot_map_merge_tests)/launch/singleRobot.launch" >
    <arg name="base" value="kobuki" />
    <arg name="stacks" value="hexagons" />
    <arg name="model" value="turtlebot1"/>
    <arg name="init_pose" value="-x 0 -y 0 -z 0"/>
  </include>
 </group>

 <group ns="turtle2">
  <param name="tf_prefix" value="turtle2" />
  <include file="$(find multirobot_map_merge_tests)/launch/singleRobot.launch" >
    <arg name="base" value="kobuki" />
    <arg name="stacks" value="hexagons" />
    <arg name="model" value="turtlebot2"/>
    <arg name="init_pose" value="-x 0 -y -6.5 -z 0"/>
  </include>
 </group>
</launch>
